### PR TITLE
Visitor chooses UI language

### PR DESCRIPTION
**Pull request by dorianbuck**
PT: https://www.pivotaltracker.com/story/show/179899774

## User Story
```
As a visitor
In order to use the UI
I would like to be able to choose the UI language
```

## Tasks
- Install `react-i18next library`
- Add UI language file
- `Suspense` language rendering
- Autodetect language
- Manual language changer

## Screenshot
Skärmavbild 2021-10-21 kl  09 56 19

### CODE REVIEW

File: cypress/integration/userCanChooseUILanguage.feature.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    cy.get("[data-cy=language-selector]").should("be.visible");
+  });
+
+  describe("when the browser is set to English on render", () => {
+    beforeEach(() => {
+      cy.visit("/", {
+        onBeforeLoad(window) {
+          Object.defineProperty(window.navigator, "language", {
+            get: cy.stub().returns("en-GB").as("language"),
+          });
```
Comment (AugustEnhager) [inline cypress/integration/userCanChooseUILanguage.feature.js:20]: Really cool implementation!

File: src/components/Header.jsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 import { Link } from "react-router-dom";
 import { useSelector } from "react-redux";
-import { Menu, Dropdown } from "semantic-ui-react";
+import { Menu, Dropdown, Select } from "semantic-ui-react";
+import { useTranslation } from "react-i18next";
+import i18n from "../i18n";
 
 const Header = () => {
   const { categories } = useSelector((state) => state);
+  const { t } = useTranslation();
```
Comment (AugustEnhager) [inline src/components/Header.jsx:10]: Is it okay/a reason for this to be a single letter function?

Comment (dorianbuck) [reply src/components/Header.jsx:10]: This is according to best practice. See example below:
```js
import React from 'react'
import { useTranslation } from "react-i18next";

const Greeting = () => {
// the t key is often standardized throughout the different frameworks. 
  const { t } = useTranslation();

  return (
    <div>
      <h1>{t('title')}</h1>
      <h2>{t('sub-title')}</h2>
    </div>
  );
}
```

**Review by AugustEnhager — APPROVED** [review]

**Review by tochman — APPROVED** [review]
LGTM

**Pull request merged**